src/main/java/com/thoughtworks/go/scm/plugin/model/requestHandlers/SCMConfigurationRequestHandler.java

```diff
@@ -13,8 +13,10 @@
     public GoPluginApiResponse handle(GoPluginApiRequest apiRequest) {
         Map<String, Object> response = new HashMap<>();
         response.put("url", createField("URL", null, true, true, false, "0"));
-        response.put("branch", createField("Branch", "master", true, false, false, "1"));
-        response.put("path", createField("Path", null, true, true, false, "2"));
+        response.put("username", createField("Username", null, true, false, false, "1"));
```

Comment:
@selvakn, we can leverage the the go-server ssh key to authenticate instead of accepting username and password.

We can support both ssh based and username/password based authentication If you feel the need for it